cli/src/cli_util.rs

```diff
@@ -1070,8 +1070,9 @@ impl WorkspaceCommandHelper {
         self.commit_summary_template().format(commit, formatter)
     }
 
-    pub fn check_rewritable<'a>(
+    pub fn check_repo_rewritable<'a>(
```

Comment:
nit: let's make it private. (this kind of API is a bit unclear because `self` also has a `repo`.)